var/spack/repos/builtin/packages/arbor/package.py

```diff
@@ -53,20 +58,23 @@ class Arbor(CMakePackage, CudaPackage):
     )
 
     # https://docs.arbor-sim.org/en/latest/install/build_install.html#compilers
-    conflicts("%gcc@:8.3")
-    conflicts("%clang@:7")
+    conflicts("%gcc@:8")
+    conflicts("%clang@:9")
     # Cray compiler v9.2 and later is Clang-based.
     conflicts("%cce@:9.1")
     conflicts("%intel")
 
-    depends_on("cmake@3.12:", type="build")
+    depends_on("cmake@3.19:", type="build")
```

Comment:
really necessary?

Reply:
I can't recall in which of the many CMake troubleshooting sessions this came up, but it was an oversight that it wasn't bumped last release, we had not been testing below 3.18 then either. If you want me to dig, I can find out what exactly we use. Does it present a problem for you? (If so, please file an issue.)